sdks/python/tox.ini

```diff
@@ -64,8 +64,9 @@ commands = false {envname} is misconfigured
 
 [testenv:py{37,38,39,310}]
 commands =
+  pip install -r build-requirements.txt
```

Comment:
2 questions:
1) Can we use deps like https://github.com/apache/beam/blob/ebb49fd9a52d07684bcc088ce024cdd9dc6b8cda/sdks/python/tox.ini#L42
2) If we're no longer automatically installing build-requirements.txt, do we need this on all of our testenv sections?

Reply:
sdist is getting build in a different environment compared to the `deps`.  I can remove that step and see if this works as expected.

Reply:
> sdist is getting build in a different environment compared to the deps

Ah, gotcha - thanks.

> I can remove that step and see if this works as expected.

I'd prefer we try that. I still don't quite get why this is necessary only for this test-env

Reply:
Yep, we don't need this actually since we automatically install build-requirements. sorry for the confusion. I removed the line